pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -221,6 +222,7 @@ func (configgen *ConfigGeneratorImpl) buildOutboundClusters(env *model.Environme
 				direction:       model.TrafficDirectionOutbound,
 				proxy:           proxy,
 				serviceMTLSMode: serviceMTLSMode,
+				mtlsReady:       service.MTLSReady,
```

Comment:
nuke this. in line 200, if Mesh external service, set the servicemTLSmode to disabled, else set it to unknown and then raed from the authn_v1alpha1_applier...

Reply:
removed with service level mtls logic